sympy/geometry/tests/test_curve.py

```diff
@@ -99,3 +99,59 @@ def test_length():
 
     c3 = Curve((t ** 2, t), (t, 2, 5))
     assert c3.length == -sqrt(17) - asinh(4) / 4 + asinh(10) / 4 + 5 * sqrt(101) / 2
+
+
+def test_tangent():
+    t = Symbol('t', real=True)
+
+    c1 = Curve((t,), (t, 0, 1))
+    assert c1.tangent == Matrix([[1]])
+
+    c2 = Curve((t, 3 * sin(t), 3 * cos(t), t ** 2), (t, 0, 1))
+    assert c2.tangent == Matrix([[1], [3 * cos(t)], [-3 * sin(t)], [2 * t]])
+
+
+def test_normal():
+    t = Symbol('t', real=True)
+
+    c1 = Curve((t,), (t, 0, 1))
+    assert c1.normal == Matrix([[0]])
+
+    c2 = Curve((t, t ** 2, cos(t), sin(t)), (t, 0, 1))
+    assert c2.normal == Matrix([[0], [2], [-cos(t)], [-sin(t)]])
+
+
+def test_binormal():
+    t = Symbol('t', real=True)
+
+    c1 = Curve((t, t, t), (t, 0, 1))
+    assert c1.binormal == Matrix([[0], [0], [0]])
+
+    c2 = Curve((1, t, t ** 2), (t, 0, 1))
+    assert c2.binormal == Matrix([[2], [0], [0]])
+
+    raises(ValueError, lambda: Curve((t,), (t, 0, 1)).binormal)
+
+
+def test_curvature():
+    t = Symbol('t', real=True)
+
+    c1 = Curve((t,), (t, 0, 1))
+    assert c1.curvature == 0
+
+    c2 = Curve((t, t ** 2), (t, 0, 1))
+    assert c2.curvature == 2 / (4 * t ** 2 + 1) ** (sympify(3) / 2)
```

Comment:
You don't need to use `sympify`, you can use `S` here. And also remove the `sympify` import at the top of the file.